locales/en.ts

```diff
@@ -1790,4 +1790,32 @@ export default {
     remove_from_favorites: "Remove from favorites",
     favorites: "Favorites",
   },
+  heatmap: {
+    week_days_short: {
+      sunday: "S",
+      monday: "M",
+      tuesday: "T",
+      wednesday: "W",
+      thursday: "T",
+      friday: "F",
+      saturday: "S",
+    },
+    month_names_short: {
+      january: "Jan",
+      february: "Feb",
+      march: "Mar",
+      april: "Apr",
+      may: "May",
+      june: "Jun",
+      july: "Jul",
+      august: "Aug",
+      september: "Sep",
+      october: "Oct",
+      november: "Nov",
+      december: "Dec",
+    },
+    no_workout: "No workout",
+    one_workout_unit: "workout",
+    multiple_workouts_unit: "workouts",
```

Comment:
You can do the same pattern provided directly by `next-international`. 

Search : 
```
    "set#zero": "set",
    "set#one": "set",
    "set#other": "sets",
```